tasks.py

```diff
@@ -860,9 +860,9 @@ class TestDependencies(NamedTuple):
 
 
 def _add_all_backends_marker(marker_string: str) -> bool:
-    # We should generalize this, possibly leveraging MARKER_DEPENDENDENCY_MAP, but for now
+    # We should generalize this, possibly leveraging MARKER_DEPENDENCY_MAP, but for now
     # right I've hardcoded all the containerized backend services we support in testing.
-    return marker_string in ["postgresql", "mssql", "mysql", "trino"]
+    return marker_string in ["postgresql", "mssql", "mysql", "spark", "trino"]
```

Comment:
Added spark here because some of the tests run by `test_case_runner_v3_api` in `test_expectations_v3_api.py` specify a spark backend. This test runs against different backends and requires the `all_backends` marker to do this. I didn't want spark to get skipped so I added it here.

Reply:
Thank you. I sounds right to me.